pygeos/constructive.py

```diff
@@ -50,6 +50,9 @@ def boundary(geometry, **kwargs):
     ----------
     geometry : Geometry or array_like
         This function will return None for geometrycollections.
+    **kwargs : dict
+        For the keyword-only arguments, see the
```

Comment:
```suggestion
        For the additional keyword-only arguments, see the
```

Reply:
I made it "For other keyword-only", exactly how numpy does it.